a11ym

```diff
@@ -41,6 +42,11 @@ program
         __dirname + '/a11ym_output'
     )
     .option(
+        '-p, --concurrency <concurrency>',
```

Comment:
First I would have choose `-c` but it's already take for `--filter-by-codes`. I will update that.